packages/app-desktop/gui/NoteSearchBar.jsx

```diff
@@ -139,9 +139,10 @@ class NoteSearchBarComponent extends React.Component {
 			color: theme.colorFaded,
 			backgroundColor: theme.backgroundColor,
 		});
-		const matchesFoundString = (query.length > 0 && this.props.resultCount > 0) ? (
+
+		const matchesFoundString = (query.length > 0 && this.props.resultCount >= 0) ? (
```

Comment:
Under which condition is this ever not true?

Reply:
so, `query.length > 0` will be not true when there is no search query, so there is no need to display 0/0.

as for `this.props.resultCount >= 0`, I just realised that this not need, so I'll remove it.